### PR TITLE
Added cluster and volume utilization details

### PR DESCRIPTION
**Pull request by shtripat**
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
Verified the changes and utilization details are listed as below in etcd

```
{

    "action": "get",
    "node": {
        "key": "/clusters/f4b4ea0d-25a7-459c-a19c-14460041d23f/Utilization",
        "dir": true,
        "nodes": [
            {
                "key": "/clusters/f4b4ea0d-25a7-459c-a19c-14460041d23f/Utilization/raw_capacity",
                "value": "52181536768",
                "modifiedIndex": 25750,
                "createdIndex": 25750
            },
            {
                "key": "/clusters/f4b4ea0d-25a7-459c-a19c-14460041d23f/Utilization/usable_capacity",
                "value": "52181536768",
                "modifiedIndex": 25751,
                "createdIndex": 25751
            },
            {
                "key": "/clusters/f4b4ea0d-25a7-459c-a19c-14460041d23f/Utilization/used_capacity",
                "value": "2119831552",
                "modifiedIndex": 25752,
                "createdIndex": 25752
            }
        ],
        "modifiedIndex": 23354,
        "createdIndex": 23354
    }
}
```

```
{

    "action": "get",
    "node": {
        "key": "/clusters/f4b4ea0d-25a7-459c-a19c-14460041d23f/Volumes/f2b420d3-39cd-458c-b6f2-7c1593f552b9/Utilization",
        "dir": true,
        "nodes": [
            {
                "key": "/clusters/f4b4ea0d-25a7-459c-a19c-14460041d23f/Volumes/f2b420d3-39cd-458c-b6f2-7c1593f552b9/Utilization/usable_capacity",
                "value": "52181536768",
                "modifiedIndex": 24670,
                "createdIndex": 24670
            },
            {
                "key": "/clusters/f4b4ea0d-25a7-459c-a19c-14460041d23f/Volumes/f2b420d3-39cd-458c-b6f2-7c1593f552b9/Utilization/used_capacity",
                "value": "2119462912",
                "modifiedIndex": 24671,
                "createdIndex": 24671
            },
            {
                "key": "/clusters/f4b4ea0d-25a7-459c-a19c-14460041d23f/Volumes/f2b420d3-39cd-458c-b6f2-7c1593f552b9/Utilization/vol_id",
                "value": "f2b420d3-39cd-458c-b6f2-7c1593f552b9",
                "modifiedIndex": 24672,
                "createdIndex": 24672
            }
        ],
        "modifiedIndex": 24523,
        "createdIndex": 24523
    }
}
```

**Comment by shtripat** [comment]
@Tendrl/tendrl-core plz review

File: tendrl/gluster_integration/objects/definition/gluster.py [hunk 1]
```diff
@@ -38,6 +38,33 @@
...
+          help: Usable capacity
+          type: int
+        used_capacity:
+          help: Used capacity
+          type: int
+      enabled: true
+      list: clusters/$TendrlContext.integration_id/Utilization
+      value: clusters/$TendrlContext.integration_id/Utilization
+      help: Cluster utilization
+    VolumeUtilization:
```
Comment (nthomas-redhat) [inline tendrl/gluster_integration/objects/definition/gluster.py]: Let us do this inline with what we are doing for ceph-integration

Comment (shtripat) [inline tendrl/gluster_integration/objects/definition/gluster.py]: Done. Added fields `usable_capacity` and `used_capacity` at volume entity level only.

**Pull request merged**